app/models/keyword.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class Keyword
+  attr_accessor :id, :name
```

Comment:
```suggestion
  include ActiveModel::Model

  attr_accessor :id, :name
```

Magic happens when you include `include ActiveModel::Model` 🚀 

You can then use it this way in the controller:

```
      def index
        first_keyword = Keyword.new({ id: 1, name: 'Test' })
        second_keyword = Keyword.new({ id: 2, name: 'Test' })
```

Reply:
Recommended reading about this: https://guides.rubyonrails.org/active_model_basics.html

Reply:
How do we decide between `ActiveModel::API` and `ActiveModel::Model`? From my understanding, Rail 7 forward, `ActiveModel::API` is preferred?

Reply:
It seems like currently they have the exact same behavior as ActiveModel includes ActiveAPI. But in the future, ActiveModel should have more "ActiveRecord like" features, while ActiveAPI aims to stay minimalist and focused on the View helpers (i.e. not including DB related helpers).

In this case, both are fine as in the future PR we will replace this `include ActiveModel/API` with a full ActiveReccord inheritance. ;-)